scripts/build_clone_guides.rb

```diff
@@ -48,30 +47,37 @@ def directory_exists?(repo_name)
         ##################
         # DRAFT GUIDES  
         # Clone guides that are still being drafted and are only for the staging website
-        if repo_name.start_with?('draft-iguide')
-            # Clone the draft interactive guides, using the dev branch for travis and master for all other environments.
-            `git clone https://github.com/OpenLiberty/#{repo_name}.git --branch #{iguide_branch} --single-branch src/main/content/guides/#{repo_name}`
-        elsif repo_name.start_with?('draft-guide')
+        if repo_name.start_with?('draft-iguide') || repo_name.start_with?('draft-guide')
+            # Clone the draft interactive guides, using the multipane/multiPane branch for travis and master for all other environments.
             `git clone https://github.com/OpenLiberty/#{repo_name}.git --branch #{guide_branch} --single-branch src/main/content/guides/#{repo_name}`
-            # Clone the default branch if the guide_branch does not exist for this guide repo.
+
+            # Clone the draft guide using multiPane because git clone is case sensitive
+            if !(directory_exists?(repo_name)){
```

Comment:
We should really consider using `octokit` to query if the GitHub repository has a branch.  This will clean up all the errors seen in the build logs.

https://www.rubydoc.info/gems/octokit/Octokit/Client/Repositories

Reply:
Thanks, i will look into using this now.